R/utils-shiny.R

```diff
@@ -0,0 +1,28 @@
+#' Internal helper for creating grid of `valuboxes`
+#' for displaying information.
+#'
+#' @keywords internal
+create_value_box_grid = function(df) {
+  vbs = purrr::imap(as.list(df), function(value, name) {
+    bslib::value_box(
+      title = htmltools::tags$span(as.character(name), style = "font-size:10pt;"),
+      value = htmltools::tags$span(as.character(value), style = "font-size:16pt;"),
```

Comment:
issue: use `shiny::` tags.
```suggestion
      title = shiny::span(as.character(name), style = "font-size:10pt;"),
      value = shiny::span(as.character(value), style = "font-size:16pt;"),
```